src/utils/updateConfigurations.js

```diff
@@ -108,6 +108,96 @@ export const createNewDenominator = (configurations, newDenominatorInfo) => {
     return configurationsToSave
 }
 
+// create new denominator relation
+export const updateDenominatorRelations = ({
+    configurations,
+    relation,
+    updateType,
+    newCode,
+}) => {
+
+    const metaDataVersion = configurations.metaDataVersion
```

Comment:
this isn't necessary (see following comment), but in general if you are going to do something like this, you could instead do something like `const {metaDataVersion, numerators, coreIndicators} = configurations`
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment